lib/analyzer/applications/node.ts

```diff
@@ -0,0 +1,118 @@
+import * as path from "path";
+import * as lockFileParser from "snyk-nodejs-lockfile-parser";
+
+import { ScannedProjectCustom } from "../../types";
+import { FilePathToContent } from "./types";
+
+interface ManifestLockPathPair {
```

Comment:
ManifestLockPathPair has 3 items, so maybe call it ManifestLockPathTuple?

Reply:
the ```pair``` here refers to the manifest + lock files, not to the items in the interface, so tuple doesn't quite work here.